HIP/hip-904.md

```diff
@@ -1170,21 +1171,50 @@ HTS system contracts will need to support the new airdrop related transactions t
-| 0x…      | <htsSystemContractAddress>.airdropTokens(TokenTransferList[] memory tokenTransfers)          |      |
-| 0x…      | <htsTokenAddress>.cancelAirdropFT(address receiverAddress)                                   | Y    |
-| 0x…      | <htsTokenAddress>.cancelAirdropNFT(address receiverAddress, int64 serialNumber)              | Y    |
-| 0x…      | <htsSystemContractAddress>.cancelAirdrops(PendingAirdropId[] memory pendingAirdrops)         |      |
-| 0x…      | <htsTokenAddress>.claimAirdropFT(address senderAddress)                                      | Y    |
-| 0x…      | <htsTokenAddress>.claimAirdropNFT(address senderAddress, int64 serialNumber)                 | Y    |
-| 0x…      | <htsSystemContractAddress>.claimAirdrops(PendingAirdropId[] memory pendingAirdrops)          |      |
-| 0x…      | <htsTokenAddress>.rejectTokenFT()                                                            | Y    |
-| 0x…      | <htsTokenAddress>.rejectTokensNFT(int64[] serialNumber)                                      | Y    |
-| 0x…      | <htsSystemContractAddress>.rejectTokens(address[] memory ftAddresses, NftId[] memory nftIds) |      |
-| 0x…      | <hasSystemContractAddress>.setAutomaticAssociations(boolean enableAutoAssociations)          | Y    |
+New system contract functions to be included to the `IHederaTokenService` interface.
+
+| Function Selector Hash |                                                               Function Signature                                                               | HAPI Transaction |    Response    |                                 |
+|------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|------------------|----------------|---------------------------------|
+| `0x2f348119`           | `function airdropTokens(TokenTransferList[] memory tokenTransfers) external returns (int64 responseCode)`                                      | TokenAirdrop     | `ResponseCode` | The response code from the call |
+| `0x012ebcaf`           | `function cancelAirdrops(PendingAirdrop[] memory pendingAirdrops) external returns (int64 responseCode)`                                       | TokenCancel      | `ResponseCode` | The response code from the call |
+| `0x05961641`           | `function claimAirdrops(PendingAirdrop[] memory pendingAirdrops) external returns (int64 responseCode)`                                        | TokenClaim       | `ResponseCode` | The response code from the call |
+| `0xebd595e0`           | `function rejectTokens(address rejectingAaddress, address[] memory ftAddresses,  NftID[] memory nftIDs) external returns (int64 responseCode)` | TokenReject      | `ResponseCode` | The response code from the call |
+
+New system contract functions to be included for a new `IHRC904` interface.
```

Comment:
Please add a note here that the facade/proxy logic is used and as such the contract address used should be the Hedera Entity on note e.g. the HTS token address or the Account EVM address.

Reply:
Added the note.